Video.js

```diff
@@ -229,6 +230,26 @@ export default class Video extends Component {
     }
   };
 
+  _onGetLicense = (event) => {
+    if (this.props.drm && this.props.drm.getLicense instanceof Function) {
+      const data = event.nativeEvent;
+      if (data && data.spc) {
```

Comment:
`data.spc` is empty in JavaScriptCore environment 🤷‍♂  I'm checking `data.spcBase64` as workaround.

Reply:
Totally I've noticed this last Friday, I don't know why but data.spc is empty in JavaScriptCore except when JS Remote Debugger is enabled 🤔

I had to implement a fallback using the B64 version of spc

Reply:
will take a look at this, thank you so much! To be honest I always checked with debugger on, so I didn't expect this to come